gax-java/gax/src/main/java/com/google/api/gax/rpc/ClientSettings.java

```diff
@@ -241,6 +246,12 @@ public B setClock(ApiClock clock) {
       return self();
     }
 
+    /** Sets the Universe Domain to configure the resolved endpoint */
+    public B setUniverseDomain(String universeDomain) {
```

Comment:
This is probably where showcase tests can help us find issues earlier. Can you please add a few showcase tests to verify that universe domain set through `ClientSettings` and `StubSettings` can be used for resolving endpoints?

Reply:
We've decided to park this. I'll create an issue to investigate aligning the stubsettings and clientsettings.